### DISCUSSION TITLE
What is minizip 1.2.11?

### DISCUSSION THREAD

**Discussion by Coeur** [nmoinvaz/minizip#97] [Q&A] [answered]
On brew, I get this log:

```
==> Upgrading minizip 
==> Downloading https://homebrew.bintray.com/bottles/minizip-1.2.11.sierra.bottl
######################################################################## 100.0%
==> Pouring minizip-1.2.11.sierra.bottle.1.tar.gz
==> Caveats
Minizip headers installed in 'minizip' subdirectory, since they conflict
with the venerable 'unzip' library.
==> Summary
🍺  /usr/local/Cellar/minizip/1.2.11: 13 files, 358.1KB

```

But on the repo, [the latest release](https://github.com/nmoinvaz/minizip/releases) is just 1.1. Could we at least tag what is version 1.2.11?

**Comment by nmoinvaz** [comment, marked as answer]
1.2.11 probably refers to the version of zlib, not the version of Minizip. Also see issue https://github.com/nmoinvaz/minizip/issues/77.

**Comment by Coeur** [comment]
OK. So not much we can do about this, except for changing `#define MINIZIP_VERSION_MINOR` and `#define MINIZIP_VERSION_PATCH`.
